src/otx/core/data/transform_libs/mmcv.py

```diff
@@ -66,10 +66,22 @@ def _check_mandatory_transforms(
     @classmethod
     def generate(cls, config: SubsetConfig) -> list[Callable]:
         """Generate MMCV transforms from the configuration."""
-        transforms = [
-            cls.get_builder().build(convert_conf_to_mmconfig_dict(cfg))
-            for cfg in config.transforms
-        ]
+        transforms = []
+        for cfg in config.transforms:
+            builder = cls.get_builder()
+            converted_cfg = convert_conf_to_mmconfig_dict(cfg)
+
+            build_cfg_type = builder.get(converted_cfg.type)
+            # TODO(sungmanc): Need to consider the mmpretrain case
+            # Both MMPretrain and MMCV have `LoadImageFromFile` transform
+            # So, There is an conflict at the registering registry.
+            # https://github.com/open-mmlab/mmpretrain/blob/main/mmpretrain/datasets/transforms/__init__.py
+            # https://github.com/open-mmlab/mmdetection/blob/main/mmdet/datasets/transforms/__init__.py
+            if build_cfg_type.__name__ == "LoadImageFromFile":
+                build_transform = builder.parent.build(converted_cfg)
+            else:
+                build_transform = builder.build(converted_cfg)
+            transforms.append(build_transform)
```

Comment:
Did you try this approach? https://github.com/openvinotoolkit/training_extensions/pull/2670#discussion_r1405969794

I don't like this since `MMCVTransformLib` is a parent of `MMPretrainTransformLib` but you are trying to anticipate the children's problem in advance, to elevate it to the parent, and solve it in the parent side.

Reply:
I tried that, if so we also need to change the function below, https://github.com/openvinotoolkit/training_extensions/blob/e538298e8b4d0cc47c636985aa46eec331226c76/src/otx/core/data/transform_libs/mmcv.py#L53

@harimkang might want to what you suggested, so, I'll change

Reply:
My suggestion was just to replace `MMPretrain.TRANSFORMS.LoadImageFromFile` with `otx.core.data.transform_libs.mmcv.LoadImageFromFile` by registering it again with `force=True` in `otx/core/data/transform_libs/mmpretrain.py`. If so, is it impossible to pass this line?
https://github.com/openvinotoolkit/training_extensions/blob/e538298e8b4d0cc47c636985aa46eec331226c76/src/otx/core/data/transform_libs/mmcv.py#L74-L77

Reply:
It works well, thanks. https://github.com/openvinotoolkit/training_extensions/pull/2678/commits/d470e5764a12f826d99beb64dfb61cc05b1c0fdd